src/main/java/org/fedoraproject/javapackages/validator/util/BytecodeVersionJarValidator.java

```diff
@@ -33,26 +34,10 @@ public void acceptJarEntry(RpmPackage rpm, CpioArchiveEntry rpmEntry, byte[] con
                 var classPath = Paths.get(jarEntry.getName());
 
                 if (classPath.toString().endsWith(".class")) {
```

Comment:
I really prefer to keep using the `var` restricted keyword unless there is a need to upcast the type due to different code.

Reply:
Fine.